jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ManifestAndConfigTemplate.java

```diff
@@ -43,13 +43,36 @@
   @SuppressWarnings("unused")
   private ManifestAndConfigTemplate() {}
 
+  /**
+   * Creates an instance.
+   *
+   * @param manifest the manifest
```

Comment:
maybe "the registry manifest"?

Reply:
I think "the image manifest" is more suitable.